docs/usage/list.rst.inc

```diff
@@ -28,6 +28,10 @@ borg list
       --umask M             set umask to M (local and remote, default: 0077)
       --remote-path PATH    set remote path to executable (default: "borg")
       --short               only print file/directory names, nothing else
+      --list-format FORMAT
+                            customize list archive file/directory listing
+                            (default: "{mode} {user:6} {group:6} {size:8d} {isomtime} {path}{extra}{NEWLINE}")
+                            Special "{formatkeys}" exists to list available keys
```

Comment:
it seems you edited this manually - this is not how this works, this is automatically generated from the argparser (see development docs).

i usually update the .inc files before release so no need to commit the changes.